lib/modules/dosomething/dosomething_signup/includes/Signup.php

```diff
@@ -110,7 +110,7 @@ private function build($data) {
       'id' => isset($northstar_user) ? $northstar_user->id : $data->field_northstar_id_value,
       'first_name' => dosomething_helpers_extract_field_data($drupal_user->field_first_name),
       'last_initial' => $last_initial,
-      'photo' => dosomething_helpers_extract_field_data($drupal_user->photo),
+      'photo' => isset($drupal_user->picture) ? $drupal_user->picture : NULL,
```

Comment:
or @deadlybutter, should we just change back to `'photo' => dosomething_helpers_isset($northstar_user, 'photo')` instead of your change [here](https://github.com/DoSomething/phoenix/commit/2a79a94b2e8760f5eba04576faa5b61099474385)?